lib/logstash/outputs/elasticsearch/protocol.rb

```diff
@@ -86,11 +104,8 @@ def bulk(actions)
             next { action => args }
           end
         end.flatten)
-        if bulk_response["errors"]
-          return {"errors" => true, "statuses" => bulk_response["items"].map { |i| i["status"] }}
-        else
-          return {"errors" => false}
-        end
+
+        simple_bulk_response(bulk_response)
```

Comment:
hmm can we call this `normalize_bulk_response` .. not sure `simple_bulk_response` is descriptive